src/query/api/v1/handler/prometheus/native/read_common.go

```diff
@@ -49,7 +49,7 @@ func read(
 	w http.ResponseWriter,
 	params models.RequestParams,
 	instrumentOpts instrument.Options,
-) ([]*ts.Series, error) {
+) ([]*ts.Series, bool, error) {
```

Comment:
nit: Should we perhaps just make the first return value here be a struct?  Then if we need to add other metadata later it should be easier both on the caller and callee side.

i.e.
```go
type readResult struct {
  series []*ts.Series
  exhaustive bool
}

func read(...) (readResult, error) { ... }
```